File: 2.0/js/texts/getTextBroadcastTexts.js
'strict'

const CallfireClient = require('callfire-api-client-js');
const client = new CallfireClient('api-login', 'api-password');

client.ready(() => {
    client.texts.getTextBroadcastTexts({
      id: 11646003,
      // get texts assigned to particular contact batch
      batchId: 5500030002,
      // search offset
      offset: 0,
      // return 10 items per request
      limit: 10,
      // return only specific fields
      fields: 'items(fromNumber,toNumber,state,message)'
    })
      .then((response) => {
        console.log(response.obj);
      })
      .catch(function (err) {
        console.log('request error ' + err.data);
      });
  },
  (clientError) => {
    console.log('client error ' + clientError);
  }
);

File: 2.0/python/texts/getTextBroadcastTexts.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.getTextBroadcastTexts(
    id=11646003,
    # get texts assigned to particular contact batch
    batchId=5500030002,
    # search offset
    offset=0,
    # return 10 items per request
    limit=10,
    # return only specific fields
    fields='items(fromNumber,toNumber,state,message)'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)